src/udpsrc.rs

```diff
@@ -556,7 +573,12 @@ impl UdpSrc {
             )
         })?;
 
-        let socket = Socket::new(&element.clone().upcast(), socket, buffer_pool);
+        let mut socket = Socket::new(
+            &element.clone().upcast(),
```

Comment:
This could nowadays be `element.upcast_ref()` :)

Reply:
in my case, an error occurs as shown below. could you advise me? 

```
error[E0599]: no method named `upcast_ref` found for type `&gst_plugin::element::Element` in the current scope
   --> src/udpsrc.rs:577:21
    |
577 |             element.upcast_ref(),
    |                     ^^^^^^^^^^
```

Reply:
Run `cargo update`, it was recently added to glib